v2/brigadier-polyfill/src/jobs.ts

```diff
@@ -39,7 +39,7 @@ export class Job extends BrigadierJob {
           spec: {
             primaryContainer: this.primaryContainer,
             sidecarContainers: this.sidecarContainers,
-            timeoutSeconds: this.timeout,
+            timeoutDuration: this.timeout + "ms",
```

Comment:
Is the `+ "ms"` here because the Brigadier `Job` type specifies timeout in ms?

```
/**
   * The duration, in milliseconds, after which Brigade should automatically
   * terminate and fail the job if it has not completed. The default is 15
   * minutes.
   */
  public timeout: number = defaultTimeout
```

Do you think we should consider changing that? It was a choice that, iirc, was inherited from Brigadier 1.x.

It does seem a bit onerous, to me, to ask people to think/work in milliseconds when that level of precision probably isn't required and also cannot be guaranteed.

Reply:
I'd be in support of changing, for sure.  Seconds, perhaps?

Reply:
That. I think seconds would be great.

Units like 1m or 24h seem like they'd be great too, but idk that JS/TS have an easy/convenient method of easily converting those to/from numeric values in a way that's consistent with Go and I definitely wouldn't want user to lose the ability to do their own calculations on that value. So sticking with a `Number` seems best and seconds seems like a reasonable unit.

Should the field's name maybe change to `timeoutSeconds` to eliminate all ambiguity?

Reply:
Clarifying:

> Should the field's name maybe change to timeoutSeconds to eliminate all ambiguity?

I'm talking about the field in the TS Job type.

On the Go end, `TimeoutDuration` is fine.